### ISSUE TITLE
Re-encoding video to WebM?

### ISSUE THREAD

**Issue by donmccurdy** [zeh/ffmpeg.app#21]
One of my more common tasks with ffmpeg is converting a video to webm for smaller size on the web. Would you be open to reviewing a PR adding this command to the app, if I wrote one up? Thanks!

**Comment by megapro17** [comment]
it will be very slow and not effective

**Comment by zeh** [comment]
> Would you be open to reviewing a PR adding this command to the app, if I wrote one up? Thanks!

Of course! FFmpegwasm [seems to be compiled with libvpx and libvorbis support](https://ffmpegwasm.netlify.app/docs/overview#libraries), so this means webm should be supported.

You can edit [/src/utils/commands/commands.json](https://github.com/zeh/ffmpeg.app/blob/dev/src/utils/commands/commands.json) and open a PR against the `dev` branch (default). That should be all that's needed to create new commands. I assume WebM encoding would be a new one, with whatever parameters and selectors you deem necessary.

I'll document the JSON's syntax and contribution workflow better in the future, but feel free to play around with it already if you have time and interest.

BTW, we already have a "Re-encode video as H.264" command in our list, but it uses some H.264 specific parameters. So I would recommend against trying to add a codec selector to make this a generic "Re-encode video as H.264 or WebM". Maybe we can do that in the future when we have more complex selector combinations/checkboxes/etc, but not yet.
